ta/mk/build-user-ta.mk

```diff
@@ -66,7 +66,7 @@ additional-compile-deps := $(ta_dev_kit-files-include)
 include mk/compile.mk
 # Install TA libraries before in-tree TAs can be linked
 additional-link-deps := $(ta_dev_kit-files-lib)
-include  ta/arch/$(ARCH)/link.mk
+include  ta/arch/link.mk
```

Comment:
This does not make much sense does it? If it's under `arch` then it applies to some specific architecture.
Same for `link_shlib.mk` and `user_ta_header.c`. I suggest moving them higher up into `ta/`.

Reply:
> This does not make much sense does it? If it's under `arch` then it applies to some specific architecture. Same for `link_shlib.mk` and `user_ta_header.c`. I suggest moving them higher up into `ta/`.

done